src/inspector_socket_server.cc

```diff
@@ -241,11 +251,26 @@ void PrintDebuggerReadyMessage(
 
 InspectorSocketServer::InspectorSocketServer(
     std::unique_ptr<SocketServerDelegate> delegate, uv_loop_t* loop,
-    const std::string& host, int port, FILE* out)
-    : loop_(loop), delegate_(std::move(delegate)), host_(host), port_(port),
-      next_session_id_(0), out_(out) {
+    const std::string& host, int port, const std::string& inspect_publish_uid,
+    FILE* out)
+    : loop_(loop),
+      delegate_(std::move(delegate)),
+      host_(host),
+      port_(port),
+      publish_uid_http_(false),
+      publish_uid_stderr_(false),
+      next_session_id_(0),
+      out_(out) {
   delegate_->AssignServer(this);
   state_ = ServerState::kNew;
+  std::vector<std::string> uid_destinations =
+      SplitString(inspect_publish_uid, ',');
+  for (const std::string& destination : uid_destinations) {
```

Comment:
Please move this to `node_options.cc` if possible. If not possible - do this in `inspector_agent.cc`.

Reply:
done.